### ISSUE TITLE
Stakeholder Analysis and User Stories

### ISSUE THREAD

**Issue by slifty** [BadIdeaFactory/truthgoggles#3]
There are a variety of types of user that Truth Goggles could try to target, and I suspect that each user story will require a slightly (or dramatically) different UI / UX treatment to have an impact.

For instance, a person who is actively interested in combating their own biases may benefit from a different type of experience than someone who is passionately supportive of their favorite politician, or who feels threatened by the direction of national politics.

Who are our target users?  What is the ecosystem they exist in / who are the players (stakeholders)  involved?

Answering these questions will make it easier to combine our lessons learned from the reading in #2 into a tool that has a shot at impact.

**Comment by slifty** [comment]
This will be done as a group activity; we could even invite other members of BIFFUD to participate if there is interest.

**Comment by knowtheory** [comment]
So this process did end up factoring into our presentation and further discussion and it's worth documenting here.

Initially we had discussed using the framework laid out in [Wardle & Derakhshan](https://firstdraftnews.org/coe_infodisorder/) as a way to frame a variety of different possible users, and the circumstances they're in.  I had complained about that i thought the model felt incomplete despite being not-wrong.  But we had issues trying to come up with ways to talk about different circumstances people find themselves in, and what action could be taken to assist in any of the contexts.

The thing Carolyn & I brought to the presentation was a cognitive framework in my case, and a population framework.  I feel way more comfortable talking about the cognitive framework and fitting in the various circumstances people find themselves in, what experiences are mediated by, and what sorts of effects they have on people.

So basically, like, we should be able to hybridize all of these into a model that lets us talk about how we can build assistive devices.
